core-data/models.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { find } from 'lodash';
+
+const models = [
+	{ name: 'postType', kind: 'root', primaryKey: 'slug', baseUrl: '/wp/v2/types' },
+];
+
+export function getModel( kind, name ) {
```

Comment:
I think you'd at one point floated the idea to me that this configuration could live in state, which would enable others to more easily extend with their own models, and might make it align better with existing concepts like getting via selector, not the standalone utility function here.